awx/settings/defaults.py

```diff
@@ -783,6 +783,11 @@
 TERRAFORM_INSTANCE_ID_VAR = 'id'
 TERRAFORM_EXCLUDE_EMPTY_GROUPS = True
 
+# ------------------------
+# OpenShift Virtualization
+# ------------------------
+OPENSHIFT_VIRTUALIZATION_EXCLUDE_EMPTY_GROUPS = True
```

Comment:
where is this being used?

Reply:
Can't find the back tick on my phone 🤣 so u gotta deal with this barf sorry 

post_run_hook expect it to exist 


tools_awx_1       | 2024-06-14 15:39:23,419 INFO     [63b8bfcc] awx.analytics.job_lifecycle inventoryupdate-8 post run {"type": "inventoryupdate", "task_id": 8, "state": "post_run", "work_unit_id": "5x2MdEm4", "task_name": "kubevirt - kubevirt"}
tools_awx_1       | 2024-06-14 15:39:23,424 ERROR    [63b8bfcc] awx.main.tasks.jobs inventory_update 8 (running) Post run hook errored.
tools_awx_1       | Traceback (most recent call last):
tools_awx_1       |   File "/awx_devel/awx/main/tasks/jobs.py", line 637, in run
tools_awx_1       |     self.post_run_hook(self.instance, status)
tools_awx_1       |   File "/awx_devel/awx/main/tasks/jobs.py", line 1669, in post_run_hook
tools_awx_1       |     if getattr(settings, '%s_EXCLUDE_EMPTY_GROUPS' % src.upper()):
tools_awx_1       |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
tools_awx_1       |   File "/awx_devel/awx/conf/settings.py", line 540, in __getattr_without_cache__
tools_awx_1       |     return getattr(self._wrapped, name)
tools_awx_1       |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
tools_awx_1       |   File "/awx_devel/awx/conf/settings.py", line 441, in __getattr__
tools_awx_1       |     return self._get_default(name)
tools_awx_1       |            ^^^^^^^^^^^^^^^^^^^^^^^
tools_awx_1       |   File "/awx_devel/awx/conf/settings.py", line 415, in _get_default
tools_awx_1       |     return getattr(self.default_settings, name)
tools_awx_1       |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
tools_awx_1       | AttributeError: 'Settings' object has no attribute 'OPENSHIFT_VIRTUALIZATION_EXCLUDE_EMPTY_GROUPS'